Paws4Me/Paws4Me/Features/Sign Up User/View Model/SignUpViewModel.swift

```diff
@@ -55,6 +57,20 @@ class SignUpViewModel {
         return isUserPasswordEmailEmpty
     }
 
+    func addUserToFirebase(name: String, surname: String, cellphone: String, address: String) {
```

Comment:
Each parameter on its own line 😄 please apply throughout the PR.

Reply:
Will do so 👍🏻